ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/manage/[ingestKeys]/CreateKeyButton.tsx

```diff
@@ -75,13 +78,28 @@ export default function CreateKeyButton() {
 
   return (
     <>
-      <Button
-        icon={<RiAddLine />}
-        btnAction={() => setModalOpen(true)}
-        disabled={!currentContent}
-        kind="primary"
-        label={`Create ${currentContent.name}`}
-      />
+      <OptionalTooltip
```

Comment:
I changed the pattern a bit for these actions when the env is disabled. Instead of disappearing them I disable them with a message to the user about why they are disabled.